src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, FigureDraw {
+    private String color;
+    private String figureName;
```

Comment:
we don't need this field. We can achieve similar in classes that extend Figure using `this.getClass().getSimpleName()`